src/theme/HntToDcSimulator.jsx

```diff
@@ -0,0 +1,107 @@
+import { PythConnection } from '@pythnetwork/client'
+import { Connection, PublicKey } from '@solana/web3.js'
+import React, { useEffect, useState } from 'react'
+import styles from "./../css/HntToDcSimulator.module.css"
```

Comment:
nit - I'd collocate this at the same place as this jsx file. The styles will not apply to any other files so there is no need to have it in a shared /css folder.